documentation/_components/activity_log.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: component
+type: component
+title: Activity log
+---
+
+<div class="activity-log">
```

Comment:
@msecret What do you think about having the `<ul>` inside of an element with `.activity-log`? Do you think it should actually just be a class that is applied to the `<ul>` itself?